src/main/java/frc/robot/constants/ClimbPositions.java

```diff
@@ -3,7 +3,7 @@
 public enum ClimbPositions {
     DOWN(0, 0),
     MID(0.5, 0.5),
-    UP(3, 3);
+    UP(4, 4);
 
     public double leftPosition;
```

Comment:
same as above, these should also include what units these positions are in such as `leftPositionMeters` or something